awx/main/wsrelay.py

```diff
@@ -236,12 +237,14 @@ async def on_ws_heartbeat(self, conn):
                     if logger.isEnabledFor(logging.DEBUG):
                         logmsg = "{} {}".format(logmsg, payload)
                         logger.warning(logmsg)
-                    return
+                    continue
```

Comment:
@relrod Do you think there's any failure scenario where this results in a tight loop? Should we add a sleep before we continue?

Reply:
Bah, I'm sorry. I just clicked Github's "more context" button and see there is already a perfectly good msg to be emitted `logmsg`

`logger.warning(logmsg)` would be preferred.

Reply:
Yeah, agree with @chrismeyersfsu, if I remember correctly, this loop blocks until there's a message to act on, so I don't think there's risk of a tight loop here where we keep on iterating and doing nothing. Every iteration should mean there was a message that triggered it.

Reply:
Made the changes suggested. Remove the else and move the logger outside the condition to always be shown.